<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Viewer für Bode-Diagramme</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="plotButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>701</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="bodePlotLayout">
     <item>
      <widget class="QCustomPlot" name="magnitudePlot" native="true"/>
     </item>
     <item>
      <widget class="QCustomPlot" name="phasePlot" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>G(s)=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>620</y>
      <width>99</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Exportieren</string>
    </property>
   </widget>
   <widget class="QComboBox" name="exportComboBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>620</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>600</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bode-Diagramm exportieren als:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="numeratorLabel">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="denominatorLabel">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>590</y>
      <width>301</width>
      <height>68</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="gainMarginTextLabel">
         <property name="text">
          <string>Amplitudenrand:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="phaseMarginTextLabel">
         <property name="text">
          <string>Phasenrand:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Beurteilung der Stabilität:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="gainMarginLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="phaseMarginLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="stabilityLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>341</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <property name="spacing">
      <number>7</number>
     </property>
     <item>
      <widget class="QLabel" name="transferFunctionLabel">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Übertragungsfunktion:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>7</number>
         </property>
         <item>
          <widget class="QLabel" name="numeratorLabel_2">
           <property name="text">
            <string>Zählerpolynom:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nennerLabel">
           <property name="text">
            <string>Nennerpolynom:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>7</number>
         </property>
         <item>
          <widget class="QLineEdit" name="numeratorInput">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Koeffizienten durch Komma trennen</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="denominatorInput">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Koeffizienten durch Komma trennen</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="settingsLabel">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Darstellungsoptionen:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
       <property name="spacing">
        <number>18</number>
       </property>
       <property name="rightMargin">
        <number>120</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>7</number>
         </property>
         <item>
          <widget class="QLabel" name="minFrequencyLabel">
           <property name="text">
            <string>min. Frequenz:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="maxFrequencyLabel">
           <property name="text">
            <string>max. Frequenz:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>7</number>
         </property>
         <item>
          <widget class="QLineEdit" name="minFrequencyInput">
           <property name="text">
            <string>10e-3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="maxFrequencyInput">
           <property name="text">
            <string>10e3</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>numeratorInput</tabstop>
  <tabstop>denominatorInput</tabstop>
  <tabstop>minFrequencyInput</tabstop>
  <tabstop>maxFrequencyInput</tabstop>
  <tabstop>plotButton</tabstop>
  <tabstop>exportComboBox</tabstop>
  <tabstop>exportButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
